hybrid/cap_build.c

```diff
@@ -15,12 +15,13 @@
 //    from the first argument --- which differentiates `cheri_address_set` and
 //   `cheri_cap_build`.
 
-#include "cheriintrin.h"
-#include "include/common.h"
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
+#include "../include/common.h"
+#include "cheriintrin.h"
+
```

Comment:
This formatting is different to the rest of the samples in `hybrid/` — perhaps it needs a `clang-format`?

Reply:
I've already applied `clang-format`, and reapplying it doesn't seem to modify the file. If you mean the fact that I split the `include`s, that's just something I do naturally between "system" and "file-specific" includes, and `clang-format` didn't seem to complain. I think there's a switch in `clang-format` to modify this, but I guess we don't turn it on this repo. If you feel strongly about another formatting, I can manually amend it.

Reply:
I don't mind, just assumed it was set by `clang-format` so thought I should point it out. I checked a few examples and nearly all have the imports ordered as this file originally had them, so I'd be tempted to revert that specific change for consistency…

…_but_ I found a couple with slight deviation, so there's no _solid_ convenion. If you'd prefer them ordered this way I'm also happy keeping the change and turning the `clang-format` flag on sometime to make the others consistent with this ordering.

Reply:
I don't have strong opinion either way, just a preference for a small a workload as necessary for this PR, but happy to go implement whatever you think is best.

Reply:
You're all being very polite :p I think we should merge as-is, and I'll let @probablytom do the honours!

Reply:
Agree! Let's hit the button 👍️